src/archive/mod.rs

```diff
@@ -279,6 +279,37 @@ impl<'a> Index<'a> {
             strtab: strings,
         })
     }
+
+    // Parses Windows Second Linker Member:
+    // number of members (m):   4
+    // member offsets:          4 * m
+    // number of symbols (n):   4
+    // symbol member indexes:   2 * n
+    // followed by SysV-style string table
+    // https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#first-linker-member
+    pub fn parse_windows_linker_member(buffer: &'a [u8]) -> Result<Self> {
+        let offset = &mut 0;
+        let members = buffer.gread_with::<u32>(offset, scroll::LE)? as usize;
+        let mut member_offsets = Vec::with_capacity(members);
+        for _ in 0..members {
+            member_offsets.push(buffer.gread_with::<u32>(offset, scroll::LE)?);
+        }
+        let symbols = buffer.gread_with::<u32>(offset, scroll::LE)? as usize;
+        let mut symbol_indexes = Vec::with_capacity(symbols);
+        for _ in 0..symbols {
+            symbol_indexes.push(buffer.gread_with::<u16>(offset, scroll::LE)? as usize);
+        }
+        let mut symbol_offsets = Vec::with_capacity(symbols);
+        for i in symbol_indexes {
+            symbol_offsets.push(member_offsets[i - 1]);
```

Comment:
Could this be done in the previous loop? That is, we don't actually need to create `symbol_indexes`.

Reply:
That's right, done.